CHANGELOG.md

```diff
@@ -4,10 +4,27 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ## Unreleased
 
+
+### Added
+
+- Added `ROLE_NONE` (ID `3` by default). Now, players aren't `ROLE_INNOCENT` but `ROLE_NONE` by default.
+- Added the possibility to give Innocents the access to a custom shop (`shopeditor`)
```

Comment:
```suggestion
- Added `ROLE_NONE` (ID `3` by default)
  - Players now default to `ROLE_NONE` instead of `ROLE_INNOCENT
  - Enables the possibility to give Innocents access to a custom shop (`shopeditor`)
```

i think this is 'just' one change, especially the second point is to my knowledge directly caused by the first. so i think the suggestions fits the overall change better